Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -172,22 +200,36 @@ - (id)init {
     return self;
 }
 
+@dynamic contactStore;
```

Comment:
No need for this if you're inmplementing it directly under it. Also, `@dynamic` isn't for manually-implemented properties; it's more for "these will exist at runtime, compiler, I promise"